kustomize/components/network-policies/README.md

```diff
@@ -0,0 +1,70 @@
+# Network Policies
+
+This guide contains instructions for deploying Online Boutique with [`NetworkPolicies`](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+
+To use `NetworkPolicies` in Google Kubernetes Engine (GKE), you will need a GKE cluster with network policy enforcement enabled.
+To learn more, see the official [GKE documentation on enabling network policy enforcement](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy#enabling_network_policy_enforcement).
+You do not need to enable network policy enforcement in clusters that use [GKE Dataplane V2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2).
+
+## Deploy Online Boutique with `NetworkPolicy` objects
+
+The steps below will deploy Online Boutique with its `NetworkPolicy` resources on a Kubernetes cluster that supports `NetworkPolicy` resources.
+
+1. Manually edit the `/kustomize/kustomization.yaml` to include the `network-policy` Kustomize Component inside this directory.
+    ```
+    components:
+      ...
+      - components/network-policy
+      ...
+    ```
+
+    Alternatively, you can use `sed` to uncomment the existing `# - component/network-policies` line:
+    ```
+    sed -i '' 's/# - components\/network-policy/- components\/network-policy/' kustomize/kustomization.yaml
+    ```
+
+1. Deploy Online Boutique with the `NetworkPolicies`.
+
+    The following command will deploy Online Boutique along with its `NetworkPolicies`:
+    ```sh
+    kubectl apply -k ./kustomize/
+    ```
+
+1. Verify that the `NetworkPolicies` have been deployed.
```

Comment:
Praise: Thanks for updating all these step numbers to `1.`. Makes maintenance easier.